app/src/main/java/co/kr/sopt_seminar_30th/application/App.kt

```diff
@@ -0,0 +1,18 @@
+package co.kr.sopt_seminar_30th.application
+
+import android.app.Application
+import androidx.appcompat.app.AppCompatDelegate
+import androidx.appcompat.app.AppCompatDelegate.MODE_NIGHT_NO
+import androidx.databinding.library.baseAdapters.BuildConfig
+import dagger.hilt.android.HiltAndroidApp
+import timber.log.Timber
+
+@HiltAndroidApp
+class App : Application() {
+    override fun onCreate() {
+        super.onCreate()
+        AppCompatDelegate.setDefaultNightMode(MODE_NIGHT_NO)
+
+        Timber.plant(Timber.DebugTree())
```

Comment:
너가 팀버를 적용한 이유는 디버그 시에만 로그를 찍어보고싶어서 갖고온 것 같은데 그렇다면

```suggestion
if (BuildConfig.DEBUG) {
        Timber.plant(Timber.DebugTree())
 }
```
이렇게 적용을 해야 디버그시에만 팀버 로그가찍혀

Reply:
저도 몰랐는데 그렇더라고요 적용하겠습니다 !